src/views/Settings.vue

```diff
@@ -283,8 +303,10 @@ export default defineComponent({
   mounted() {
     this.appVersion = this.appInfo.branch ? (this.appInfo.branch + "-" + this.appInfo.revision) : this.appInfo.tag;
   },
-  ionViewWillEnter() {
-    this.getCurrentFacilityDetails()
+  async ionViewWillEnter() {
+    await this.getCurrentFacilityDetails()
+    await this.getCurrentOrdersCount()
+    this.updateOrderLimitType()
```

Comment:
I think there is no need to call this method always, it should be called only when the api for fetching the order count returns success.